### ISSUE TITLE
Color of links in block quotes same as color of normal text.

### ISSUE THREAD

**Issue by yeruvan** [nickmilo/LYT-Mode#46]
See above. Can't see links in block quotes unless you hover your mouse over them. Thanks! :)

**Comment by nickmilo** [comment]
Yes, this issue is known and quite bothersome. Hoping to figure out a solution soon.

**Comment by nickmilo** [comment]
If anyone out there can fix it, please go for it!

**Comment by W1-PopelierE** [comment]
Is this still a problem? If so, please attach a screenshot so I know where to look 😄 

On my end it looks like this:
<img width="347" alt="image" src="https://user-images.githubusercontent.com/17490821/212542921-2bbad8d2-8cbd-4498-80c4-b06868ac8893.png">

<img width="329" alt="image" src="https://user-images.githubusercontent.com/17490821/212542925-c4736c2e-1b90-4599-9036-1bc02685a1bc.png">
